src/test/java/com/scottlogic/filters/AuthorPostFilterTest.java

```diff
@@ -44,42 +48,71 @@ void filter_EmptyList_EmptyList() {
 
     @Test
     void filter_1ItemList_1ItemList() {
-        List<UserPost> inputList = List.of(userPost1);
-        List<UserPost> expected = List.of(userPost1);
+        List<UserPost> inputList = List.of(userPostJane);
+        List<UserPost> expected = List.of(userPostJane);
 
         List<UserPost> actual = new AuthorPostFilter("Jane").filter(inputList);
 
         assertEquals(expected, actual);
     }
 
+    @Test
+    void filter_1Item0Match_EmptyList() {
+        List<UserPost> inputList = List.of(userPostJane);
+        List<UserPost> expected = List.of();
```

Comment:
An interesting question here of construction of an empty list. At the moment you have made the construction similar to the construction within the function but dissimilar to constructions within the file. In general it is nice to have consistency across the file and the method but here it might be readability choice and either way is fine, both would be ideal though.

Reply:
I think Intellij advises against using Arrays.asList() for creation of empty lists hence the inconsistency

Reply:
Ahh okay, in which case go with whatever Intellij suggests, it's generally best most of the time, some of its suggestions will be readability but some are efficiency changes so are often worth listening to. However if you do have a specific reason for doing something, such as big changes in readability, then ignoring isn't going to ruin the code, for example it will often suggest changing lambdas to method functions but sometimes the method functions are so verbose they make the code unreadable, there as the efficiency difference is only 1 step so it can sometimes be worth it.